DSA_Session2/src/queueImplementation/ArrayQueueTest.java

```diff
@@ -0,0 +1,108 @@
+package queueImplementation;
+
+import static org.junit.Assert.*;
+
+import org.junit.Before;
+import org.junit.Test;
+/**
+ * this class test the ArrayQueue operations
+ * @author Aman Gautam
+ * Dated 01/08/2019
+ */
+public class ArrayQueueTest {
+	Queue queue = null;
+	@Before
+	public void Initialize(){
+		queue = new ArrayQueue(5);
+	}
+	
+	@Test
+	public void testEnQueue() {
+		queue.enQueue(1);
+		assertEquals(1,queue.queueSize());
+		
+		queue.enQueue(2);
+		assertEquals(2,queue.queueSize());
+		
+		
+		queue.enQueue(3);
+		assertEquals(3,queue.queueSize());
+			
+	}
+
+	@Test
+	public void testDeQueue(){
+		
+		assertEquals(-1111, queue.deQueue());
```

Comment:
Please verify these test case

Reply:
assertEquals(-2147483648, queue.deQueue())    : this should be the assert function since min value is -2147483648 not -1111 .